src/client/puppet-hostie.ts

```diff
@@ -222,10 +222,11 @@ export class PuppetHostie extends Puppet {
       endpoint = hostieIpResult.ip + ':' + hostieIpResult.port
     }
 
+    const clientOptions = Object.assign({}, GRPC_LIMITATION, { 'grpc.default_authority':this.options.token })
```

Comment:
Please change to use the ES6 way to implement this because it's more readable.

```ts
const clientOptions = {
  ...GRPC_LIMITATION,
  'grpc.default_authority': this.options.token,
}
```